lld/wasm/Config.h

```diff
@@ -72,7 +72,7 @@ struct Configuration {
   bool stripAll;
   bool stripDebug;
   bool stackFirst;
-  bool isStatic = false;
+  bool isStatic = true;
```

Comment:
Perhaps a comment:  "Because dyamanic linking under Wasm is still experimental we default to static linking"

Reply:
done